test/functionalTest/cases/947_v2_entities/v2_entities_simple.test

```diff
@@ -0,0 +1,426 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I was about to change the name of this functest, but my idea was to remove 'simple' (it is not simple anymore) and add GET:

947_v2_entities/GET_v2_entities.test.

Much easier to understand that it's about a "GET /v2/entities" that if we use some other name for the operation, don't you think?